samples/consul-ui/consul_services.json.erb

```diff
@@ -82,7 +82,7 @@
                  id:      snode['Service']['ID'],
                  name:    snode['Node']['Node'],
                  sMeta:   meta ? meta : {},
-                 addr:    snode.service_address,
```

Comment:
I would keep this: you are not sure that name will be resolvable using DNS

Reply:
Ah yes, it was causing a bug when executing the template and forgot to remove the comments

Reply:
Done